store/sqlstore/upgrade.go

```diff
@@ -963,6 +963,7 @@ func upgradeDatabaseToVersion532(sqlStore *SqlStore) {
 
 func upgradeDatabaseToVersion533(sqlStore *SqlStore) {
 	if shouldPerformUpgrade(sqlStore, Version5320, Version5330) {
+		sqlStore.AlterColumnTypeIfExists("Roles", "Permissions", "longtext", "text")
```

Comment:
You will want to put this in 534 not 533 (5.33 is already out).

Reply:
OK.